docs/troubleshooting/failed-schema-sync.rst

```diff
@@ -0,0 +1,11 @@
+Failed Schema Sync
+===================
+
+The total binary footprint of all ScyllaDB columns, tables, etc., must fit a single commit log segment size 
+divided by two. If this requirement is not met, schema sync may fail.
+
+In such a case, you may need to increase the commitlog segment size limit. To do this, 
+go to ``/etc/scylla/scylla.yaml`` and set the ``commitlog_segment_size_in_mb`` parameter to a higher value.
```

Comment:
This duplicates information that should otherwise be present elsewhere. Better have a systematic description of all scylla.yaml parameters, instruction how to change it and the current defaults.

Reply:
> This duplicates information that should otherwise be present elsewhere.

I agree, but it's not present anywhere, so let's add it to Troubleshooting to help the current users.

> Better have a systematic description of all scylla.yaml parameters, instruction how to change it and the current defaults.

Again, I agree, and we do have it in the backlog. However, we cannot manually add and maintain the descriptions of scylla.yaml parameters, as it would be error-prone, and the docs may easily become outdated.
It's a cross-team initiative to develop appropriate automation in order to ensure that the scylla.yaml documentation is updated across releases.

Reply:
@kostja We've resumed working on documenting the parameters. You can monitor the progress here: https://github.com/scylladb/scylladb/issues/8424